vyper/venom/basicblock.py

```diff
@@ -131,14 +153,40 @@ class IRVariable(IRValue):
     mem_addr: Optional[int] = None
 
     def __init__(
-        self, value: str, mem_type: MemType = MemType.OPERAND_STACK, mem_addr: int = None
+        self,
+        value: str,
+        mem_type: MemType = MemType.OPERAND_STACK,
```

Comment:
add a note to remove the `mem_type` stuff (unless it's going to be useful later)